pkg/martaapi/taxonomy.go

```diff
@@ -209,3 +204,23 @@ var LineDirections = map[Line][]Direction{
 	Gold:  []Direction{North, South},
 	Red:   []Direction{North, South},
 }
+
+//Termini allow for lookups of all the terminuseses of the different lines
+var Termini = map[Line]map[Direction]Station{
+	Green: map[Direction]Station{
```

Comment:
not saying this should be changed now, but this kind of universal information might best be stored somewhere more central since I could envision other repo's using this taxonomy. Maybe the gomarta client? or maybe a different repo. Thoughts?

Reply:
Yes Chris brought this up a while ago and I 100% agree, if you can make it to the meeting tomorrow let's chat about options for that